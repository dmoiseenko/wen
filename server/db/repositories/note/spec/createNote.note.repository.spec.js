const { createNote } = require('../createNote.note.repository');
const { getNotes } = require('../getNotes.note.repository');
const bootstrap = require('../../../bootstrap/bootstrap.js');


beforeEach(() => bootstrap());

it('should add new note', async () => {
  await createNote('note text');

  const actual = await getNotes();

  expect(actual).toMatchSnapshot();
});

it('should return new note', async () => {
  const actual = await createNote('note text');

  expect(actual).toMatchSnapshot();
});
