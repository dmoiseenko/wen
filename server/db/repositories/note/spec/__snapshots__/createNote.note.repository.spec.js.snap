// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add new note 1`] = `
Array [
  Object {
    "id": 1,
    "text": "Well done is better than well said. Benjamin Franklin",
  },
  Object {
    "id": 2,
    "text": "Look deep into nature, and then you will understand everything better. Albert Einstein",
  },
  Object {
    "id": 3,
    "text": "Be yourself; everyone else is already taken. Oscar Wilde",
  },
  Object {
    "id": 4,
    "text": "note text",
  },
]
`;

exports[`should return new note 1`] = `
Object {
  "id": 4,
  "text": "note text",
}
`;
